docs/src/integrate/nodejs-api.md

```diff
@@ -557,18 +557,20 @@ const messages = linter.verify(code, {
 The `verify()` method returns an array of objects containing information about the linting warnings and errors. Here's an example:
 
 ```js
-{
-    fatal: false,
-    ruleId: "semi",
-    severity: 2,
-    line: 1,
-    column: 23,
-    message: "Expected a semicolon.",
-    fix: {
-        range: [1, 15],
-        text: ";"
+[
+    {
+        fatal: false,
+        ruleId: "semi",
+        severity: 2,
+        line: 1,
+        column: 23,
+        message: "Expected a semicolon.",
+        fix: {
+            range: [1, 15],
+            text: ";"
+        }
     }
-}
+]
 ```
```

Comment:
This isn't a change in v9, I just wrapped the lint message example in `[]` because `verify()` always returns arrays.